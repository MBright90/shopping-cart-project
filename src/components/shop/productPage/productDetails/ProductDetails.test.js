import { render } from '@testing-library/react'
import React from 'react'
import { HashRouter } from 'react-router-dom'

import ProductDetails from '.'

describe('ProductDetails', () => {
  let props

  beforeEach(() => {
    props = {
      product: {
        name: 'Test Name',
        price: 100,
        description: 'Test description'
      }
    }
  })

  it('renders information to match snapshot', () => {
    const { container } = render(<ProductDetails {...props} />, {
      wrapper: ({ children }) => <HashRouter basename="/">{children}</HashRouter>
    })
    expect(container).toMatchSnapshot()
  })
})
