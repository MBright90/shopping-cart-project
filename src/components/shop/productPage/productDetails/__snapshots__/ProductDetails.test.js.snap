// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductDetails renders information to match snapshot 1`] = `
<div>
  <div
    class="productDetails"
  >
    <h1
      class="productHeadline"
    >
      <span
        class="productName"
      >
        Test Name
      </span>
       
      <span
        class="productPrice"
      >
        Â£
        100
      </span>
    </h1>
    <p
      class="productDescription"
    >
      Test description
    </p>
  </div>
</div>
`;
